modules/persistent-storage-csi-gcp-hyperdisk-storage-pools-procedure.adoc

```diff
@@ -12,6 +12,41 @@
 .Procedure
 To set up hyperdisk-balanced disks:
 
+ifdef::openshift-dedicated[]
+. Create an {product-title} cluster on {GCP} with attached disks provisioned with hyperdisk-balanced disks. This can be achieved by provisioning the cluster with compute node types that support Hyperdisk Balanced disks, such as the C3 and N4 machine series from {GCP}.
```

Comment:
```suggestion
. Create an {product-title} cluster on {GCP} with attached disks provisioned with hyperdisk-balanced disks. This can be achieved by provisioning the cluster with compute node types that support hyperdisk-balanced disks, such as the C3 and N4 machine series from {GCP}.
```